firewood/src/merkle.rs

```diff
@@ -1761,7 +1767,21 @@ mod tests {
     #[test_case(branch(b"value".to_vec(), None); "branch without chd")]
     #[test_case(branch(Vec::new(), None); "branch without value and chd")]
     fn node_encode_decode_(node: Node) {
-        let merkle = create_test_merkle();
+        let merkle = create_test_merkle::<Bincode>();
+        let node_ref = merkle.put_node(node.clone()).unwrap();
+        let encoded = merkle.encode(&node_ref).unwrap();
+        let new_node = Node::from(merkle.decode(encoded.as_ref()).unwrap());
+        let new_node_hash = new_node.get_root_hash(merkle.store.as_ref());
+        let expected_hash = node.get_root_hash(merkle.store.as_ref());
+        assert_eq!(new_node_hash, expected_hash);
```

Comment:
This is really hard to read in a big block like this. Can we add _some_ spacing? Doesn't have to be as much as I would add, any is better than none.